common/src/cursor.rs

```diff
@@ -128,7 +128,7 @@ lazy_static! {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::proto::Position;
+    use crate::{cursor, proto::Position, traversal::ensure_nearby};
```

Comment:
`crate::cursor` and `super` are the same thing in this context, so there's no need to import `crate::cursor` or qualify things with it below.

Reply:
Pushed an amend commit that addresses this.